lxc.md

```diff
@@ -123,7 +123,6 @@ Set up hostfiles on each box with hardcoded IP addresses
 
 
 ```
-127.0.0.1 localhost n1 n1.local
```

Comment:
I've addressed the `net/ip` issue, so that's fixed. But the localhost issue remains: when `127.0.0.1 localhost n1 n1.local` is removed from /etc/hosts everything works fine, but when localhost is specified in etc/hosts, the cluster machines cannot talk to each other:
```
2015/01/30 11:48:24 raft: 5440ff22fe632778 [logterm: 1, index: 5] sent vote request to 9b116f88cab4dc9 at term 2
2015/01/30 11:48:24 sender: error posting to 5aa594b5d9b66c42: dial tcp 192.168.122.12:2380: connection refused
2015/01/30 11:48:24 sender: the connection with 5aa594b5d9b66c42 becomes inactive
```

Slightly strange.